### ISSUE TITLE
Unzip Slides

### ISSUE THREAD

**Issue by AmoDinho** [AmoDinho/datacamp-python-data-science-track#3]
The slides for most of the courses need to be unzipped.

**Comment by itstalmeez** [comment]
No, if you want to download, simply download it because whenever you'll download it from GitHub it will download in .zip format
